modules/apps/headless/headless-admin-workflow/headless-admin-workflow-impl/src/main/java/com/liferay/headless/admin/workflow/internal/resource/v1_0/WorkflowLogResourceImpl.java

```diff
@@ -105,6 +105,9 @@ private String _toLogTypeName(WorkflowLog.Type type) {
 		else if (type == WorkflowLog.Type.TASK_COMPLETION) {
 			return LogType.TASK_COMPLETION.name();
 		}
+		else if (type == WorkflowLog.Type.NODE_ENTRY) {
```

Comment:
@MariMalta, could you put this conditional above the `WorkflowLog.Type.TASK_ASSIGN` conditional to keep the alphabetical order?, thank you!